src/modules/QtQuick/Image.js

```diff
@@ -15,6 +15,7 @@ QmlWeb.registerQmlType({
     asynchronous: { type: "bool", initialValue: true },
     cache: { type: "bool", initialValue: true },
     smooth: { type: "bool", initialValue: true },
+    noFlicker: { type: "bool", initialValue: false },
```

Comment:
This is not a standard property, so we shouldn't add it. We should stick to one of the behaviours by default to more accurately match the Qt behaviour.

The additional functionality could go somewhere else, but not in QtQuick module.
E.g. it could be something like `QmlWeb/Image` or `QmlWeb.Addons/Image`, though I doubt if we really need that — probably the best way would be to just do what Qt does by default.